src/AppBundle/Controller/RfXAnalysisController.php

```diff
@@ -0,0 +1,213 @@
+<?php
+/**
+ * This file contains the code that powers the RfX Analysis page of xTools.
+ *
+ * @category RfXAnalysis
+ * @package  AppBundle\Controller
+ * @author   Xtools Team <xtools@lists.wikimedia.org>
+ * @license  GPL 3.0
+ * @link     http://xtools.wmflabs.org/rfa
```

Comment:
`XTools` :) And not sure about the mailing list... is anyone checking that? In my opinion `@author` is obsolete anyway now that we have git blame/log